jupyterlab/labapp.py

```diff
@@ -897,13 +897,15 @@ def initialize(self, argv=None):
                 import jupyter_collaboration  # noqa
             except ImportError:
                 self.log.critical(
-                    """
+                    """\
+Exiting due to inability to start collaboration:
```

Comment:
I do not understand this sentence, can you try to refolmulate?

Reply:
Is there a particular part that's causing the issue?

The process is exiting.  The reason for exiting is that collaboration 1) is enabled; and 2) cannot be started.

Perhaps:


```
    """\
    Juptyer Lab cannot start, because `jupyter_collaboration` was configured but cannot be `import`ed.
    
    [rest of message]
    """"
```

Reply:
I've pushed this, in case it works

Reply:
This wording looks great to me, thank you!. Let's work on the other part too - I will leave a separate comment.